docs/inference-providers/guides/github-actions-code-review.md

```diff
@@ -0,0 +1,182 @@
+
+## Step 1: Install OpenCode
+
+First, install OpenCode on your local machine see the [installation instructions](https://opencode.ai/docs/#install).
+
+## Step 2: Configure OpenCode for GitHub
+
+Navigate to your local clone of your GitHub repository and run the setup command:
+
+```bash
+cd your-repository
+opencode github install
+```
+
+This interactive setup will guide you through:
+
+1. **Installing the GitHub app** - Your browser opens to authorize OpenCode for your repositories. You can choose specific repositories or all repositories.
+2. **Selecting a provider** - Choose **Hugging Face** from the list
+3. **Choosing a model** - Select a model like **GLM-4.5-Air** or **Kimi-K2-Instruct**
+4. **Creating the workflow** - Generates `.github/workflows/opencode.yml` automatically
+
+Once the `.github/workflows/opencode.yml` file is created, you'll need to commit and push it to your repository:
+
+```bash
+git add .github/workflows/opencode.yml
+git commit -m "Add OpenCode workflow"
+git push
+```
+
+After the setup completes, you'll need to add your Hugging Face token as a repository secret:
+
+1. Get a token from your [Hugging Face settings](https://huggingface.co/settings/tokens/new?ownUserPermissions=inference.serverless.write&tokenType=fineGrained). The token needs permissions to `Make calls to Inference Providers`.
+2. In your GitHub repository, go to **Settings → Secrets and variables → Actions**
+3. Click **New repository secret**
+4. Name it `HF_TOKEN` and paste your token
+
+> [!TIP]
+> Make sure your Hugging Face token has Inference Providers permissions enabled. We recommend creating a separate token specifically for OpenCode usage.
+
+## Step 3: Try It Out
+
+Once your workflow is set up and your token is configured, test it by commenting on any issue:
+
+```
+/oc summarize
+```
+
+OpenCode will analyze the issue and provide a summary. Here are other commands you can try:
+
+**Explain an issue:**
+
+```
+/opencode explain this issue
+```
+
+**Implement a fix:**
+
+OpenCode creates a new branch, implements the changes, and opens a pull request.
+
+**Request changes on a PR:**
+
+```
+/oc please add error handling
+```
+
+> [!TIP]
+> Use `/oc` (short) or `/opencode` (long) to trigger commands. OpenCode understands natural language, so feel free to be specific about what you need.
+
+## Security Considerations
+
+> [!WARNING]
+> On public repositories, anyone can trigger the bot by commenting `/oc` or `/opencode`. This could lead to unexpected inference costs or unwanted PRs. Consider:
+>
+> - **Using a separate Hugging Face token** specifically for OpenCode (not your main token), so you can revoke it if needed without affecting other services
+> - Monitoring your Hugging Face usage to track costs
+> - Adding workflow conditions to restrict who can trigger the bot (e.g., only repository collaborators)
```

Comment:
This sounds very logical. Perhaps we can show how, if it doesn't make it too long?